changelog.d/420.change.rst

```diff
@@ -0,0 +1 @@
+Add weakref parameter to attr.s() allowing for weak referenceable slots classes.
```

Comment:
We call them “slotted classes”: http://www.attrs.org/en/stable/glossary.html as you do later in the diff. :)

Reply:
FYI, a search actually shows several hits for `slots class`.  But yes, I'm correcting this.